src/index.ts

```diff
@@ -208,6 +253,102 @@ const main = (vortex: VortexExtensionContext) => {
     */
   }
 
+  vortex.registerReducer([`settings`, `redmod`], reducer);
+
+  vortex.registerSettings(`RedMods`, settings, undefined, () => {
+    const state = vortex.api.store.getState();
+    const gameMode = vortexApi.selectors.activeGameId(state);
+    return gameMode === GAME_ID;
+  }, 51);
+
+  // REDmod TODO
+  vortex.registerToDo(
+    `redmod-enableness`,
+    `settings`,
+    (state: VortexState): IREDmodProps => {
+      const gameMode = vortexApi.selectors.activeGameId(state);
+      return {
+        gameMode,
+        redmodEnabled: redModEnable(state),
+        autoDeployEnabled: autoRun(state),
+        archiveAutoConvertEnabled: archiveAutoConvert(state),
+      };
+    },
+    `download`,
+    `REDmod Integration`,
+    (props: IREDmodProps) => {
+      toggleREDmodIntegration(vortex.api, props.gameMode);
+      vortex.api.events.emit(`analytics-track-click-event`, `Dashboard`, `REDmodding ${props.redmodEnabled ? `ON` : `OFF`}`);
```

Comment:
This is Nexus analytics, right? Do we need it? (Or rather do we need to give it to them if we don't get it back? :)

Reply:
Heh. To be honest, I am not sure. I guess I should look at the documentation for that event specifically. Maybe we could figure out where the analytics are going and detour it a bit so we can grab it :)

Reply:
Well can always listen for the event but there's no way to get it sent :)